lib/rabbitmqctl.ex

```diff
@@ -380,11 +379,11 @@ defmodule RabbitMQCtl do
 
   defp format_error(:undef, _opts, _module) do
     {:error, ExitCodes.exit_software(),
-      "Function clause.\nStacktrace:\n" <> Exception.format_stacktrace()}
```

Comment:
FYI `Exception.format_stacktrace()` will _not_ get a useful stacktrace at this point.